ChessVariantsAPI.Tests/GameOrganizationTests/GameOrganizerTests.cs

```diff
@@ -0,0 +1,106 @@
+using ChessVariantsAPI.GameOrganization;
+using ChessVariantsLogic;
+using System;
+using Xunit;
+
+namespace ChessVariantsAPI.Tests;
+
+public class GameOrganizerTests : IDisposable
+{
+    private GameOrganizer gameOrganizer;
+
+    public GameOrganizerTests()
+    {
+        gameOrganizer = new GameOrganizer();
+        gameOrganizer.CreateGame("0", "id");
+    }
+
+    public void Dispose()
+    {
+        gameOrganizer = new GameOrganizer();
+        gameOrganizer.CreateGame("0", "id");
+        GC.SuppressFinalize(this);
+    }
+
+    [Fact]
+    public void GetGame_ShouldReturnGameIfItExists()
+    {
+        var game = gameOrganizer.GetGame("0");
+        Assert.NotNull(game); // GetGame cannot return null value, this is just to make sure no exceptions are thrown
+    }
+
+    [Fact]
+    public void GetGame_ShouldThrowExceptionIfNoGameExists()
+    {
+        Assert.Throws<GameNotFoundException>(() => gameOrganizer.GetGame("non_existing_game_id"));
+    }
+
+    [Fact]
+    public void CreateNewGame_ShouldAddGame()
+    {
+        gameOrganizer.CreateGame("1", "id");
+        var game = gameOrganizer.GetGame("1");
+        Assert.NotNull(game);
+    }
+
+    [Fact]
+    public void CreateGame_ShouldThrowExceptionIfGameAlreadyExists()
+    {
+        Assert.Throws<OrganizerException>(() => gameOrganizer.CreateGame("0", "id"));
+    }
+
+    [Fact]
+    public void JoinGame_ShouldWork()
+    {
+        gameOrganizer.JoinGame("0", "second_id");
+        Assert.True(true); // Might look weird but this test is simply to make sure no exceptions are thrown.
+    }
```

Comment:
Perhaps something like this would be a bit more clear of what is tested?

var exception = Record.Exception(() => gameOrganizer.JoinGame("0", "second_id"));
Assert.Null(exception);

Reply:
Didn't know about that Record-thing, looks much better! thanks :D